src/core_plugins/getting_started/public/components/getting_started/getting_started.html

```diff
@@ -0,0 +1,135 @@
+<div class="kuiViewContent gettingStartedContent">
+
+  <div
+    ng-if="!gettingStarted.hasOptedOut"
+    class="kuiBar gettingStartedOptOutContent"
+  >
+    <div class="kuiBarSection">
+      <p class="kuiText kuiSubduedText">
+        <a
+          kbn-href="#/management"
```

Comment:
It seems weird to drop a user in Management, where there's really nothing clear to do or read. Wouldn't we still want to funnel them to the index-pattern creation UI?